docker/jenkins-images/Dockerfile-build-base

```diff
@@ -1,17 +1,20 @@
 ARG FROM_VERSION
 FROM harbor.h2o.ai/opsh2oai/h2o-3/dev-r-3.4.1:${FROM_VERSION}
 
-RUN \
-    curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
-    apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y \
-        texlive \
-        texlive-fonts-extra \
-        texlive-htmlxml \
-        texinfo \
-        texlive-bibtex-extra \
-        texlive-formats-extra \
-        texlive-generic-extra && \
```

Comment:
`texlive-generic-extra` doesn't exist anymore. It used to be a transitional package that installed `texlive-plain-generic` which I added in the previous step when removing `texlive-htmlxml`.

See https://answers.launchpad.net/ubuntu/bionic/amd64/texlive-generic-extra/2017.20180305-1